csrankings-9.csv

```diff
@@ -908,6 +908,7 @@ Wonsook Lee,University of Ottawa,http://www.site.uottawa.ca/~wslee,ohzzc8AAAAAJ
 Wonyong Sung,Seoul National University,https://sites.google.com/site/spslab2018/professor,1IfNFz4AAAAJ
 Wonzoo Chung,Korea University,http://spl.korea.ac.kr/professor,NOSCHOLARPAGE
 Woo Jin Lee,Kyungpook National University,http://selab.knu.ac.kr,mu0cyK8AAAAJ
+Woohun Lee,KAIST,http://wonderlab.kaist.ac.kr,z0lilFMAAAAJ

```

Comment:
Website is down.

Reply:
Also, not listed here: https://cs.kaist.ac.kr/people/faculty?kind=&gubun=L&gubun_lang=en&research_area=&keyword=&radio_lang=en